website/docs/patterns/feature-highlight/feature-highlight.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Feature highlight
+---
+
+## Description
+
+You can promote certain features in your UI by using special styles from the `FeatureHighlight` package.
+
+All components from the `FeatureHighlight` package are wrappers over our standard components, such as [Button](../../components/button/button) or [Notice](../../components/notice/notice), and you can use all their standard properties.
+
+## General principles
+
+* Make sure there's no more than 1–2 highlighted features on the page at the same time.
+* Replace the highlighted control with its standard version after approximately a month of promotion.
+* If there's an animation on click, disable it after the user has activated the control several times.
+
+## Badge
+
+To add more accent to [Input](#input), [TabLine](#tabline), or other components, use the `BadgeFH` component.
+
+::: sandbox
+
+<script lang="tsx">
+  export Demo from 'stories/patterns/ux-patterns/feature-highlight/docs/examples/badge.tsx';
+</script>
+
+:::
+
+## Button
+
+To highlight a [Button](../../components/button/button), use the `ButtonFH` component.
+
+For the primary button, use the standard addon.
```

Comment:
Why didn't we add animation for our primary button? ✨ It's because it will be too much with the gradient fill that button already has?